### PR TITLE
[PRM-2778] - Made TaxReturnStatus optional for R5B changes

### PR DESCRIPTION
**Pull request by LewisMitchell**

### CODE REVIEW

**Comment by LewisMitchell** [comment]
Config PR: https://github.com/hmrc/app-config-production/pull/19297

File: app/models/getPenaltyDetails/lateSubmission/TaxReturnStatusEnum.scala [hunk 1]
```diff
@@ -22,7 +22,7 @@ object TaxReturnStatusEnum extends Enumeration {
   val Open: TaxReturnStatusEnum.Value = Value
   val Fulfilled: TaxReturnStatusEnum.Value = Value
   val Reversed: TaxReturnStatusEnum.Value = Value
-  @deprecated("Not to be used, equivalent to None (Added Frequency Adjustment Point)",  "16/05/2023")
+  @deprecated("Not to be used, equivalent to None (Added Frequency Adjustment Point)", "16/05/2023")
   val AddedFAP: TaxReturnStatusEnum.Value = Value(" ")
```
Comment (JBlacklaws) [inline app/models/getPenaltyDetails/lateSubmission/TaxReturnStatusEnum.scala:26]: Now I've removed this Enum from the frontend, I think we won't need it in the backend?

**Review by JBlacklaws — CHANGES_REQUESTED** [review]

File: app/models/getPenaltyDetails/lateSubmission/TaxReturnStatusEnum.scala (hunk 1 above)
Comment (LewisMitchell) [reply app/models/getPenaltyDetails/lateSubmission/TaxReturnStatusEnum.scala:26]: We will still read it in as a blank value until R5B is deployed in September

Comment (JBlacklaws) [reply app/models/getPenaltyDetails/lateSubmission/TaxReturnStatusEnum.scala:26]: Fair enough!

**Review by JBlacklaws — APPROVED** [review]

**Pull request merged**